isso/js/embed.js

```diff
@@ -57,6 +57,14 @@ require(["app/lib/ready", "app/config", "app/i18n", "app/api", "app/isso", "app/
             config["max-comments-top"],
             config["max-comments-nested"]).then(
             function (rv) {
+                for (var setting in rv.config) {
+                    config[setting] = rv.config[setting]
+                }
+
+                $("#isso-thread").append($.new('h4'));
```

Comment:
These snuck in by accident

Reply:
Also, would be nice if there was at least a message in the browser console about a client setting being overridden by a server setting. That'd require separating against the client defaults though.

Reply:
When you say "separating against", do you mean keeping both sets of values so as to show the differences in the console message?

Reply:
No, I mean we'd need a way to check if the user has attempted to modify a client setting via `data-isso-*` before doing this type of check:

```js
if (config[setting] != rv.config[setting]) {
    console.log("Isso: Client value xyz overridden by server value.");
    console.log("Since Isso version 0.12.6, data-isso-%value is only configured via the server to keep client and server in sync");
    config[setting] = rv.config[setting];
}
```

There might be a discrepancy just because the client is still using default values but the value has been set on the server.

Reply:
Ah... In my original commit I moved those lines into the `api.fetch` callback, because the `Postbox` constructor needed the final configuration and therefore had to be called after receiving/updating the config from the server response. The auto-merger screwed up and just added the code instead of moving it, since the code looks a bit different in master now. It still needs to be moved.

Reply:
Will fix later this week.

Reply:
Please squash while you're at it. Thanks!

Reply:
Fixed the auto-merger mistake and added the warning message based on the suggestion above (but note the extra condition).

I will wait with squashing until someone has actually run the code, in case more fixes are needed.